halo2-snark-aggregator-circuit/src/verify_circuit.rs

```diff
@@ -144,14 +144,14 @@ impl<
                             ctx,
                             &schip,
                             8usize,
-                            33usize,
+                            63usize,
                         )
                         .unwrap();
 
                     proof_data_list.push(ProofData {
                         instances,
                         transcript,
-                        key: format!("c{}p{}", ci, i),
+                        key: format!("{}_p{}", ci, i),
```

Comment:
I remember previously it is circuit name rather than circuit index here？

Reply:
fixed with https://github.com/scroll-tech/halo2-snark-aggregator/pull/35/commits/fd933c78f3dede595eaff61c9cc0505b2a94afc2